packages/portalnetwork/src/client/client.ts

```diff
@@ -171,6 +176,12 @@ export class PortalNetwork extends EventEmitter<PortalNetworkEvents> {
   constructor(opts: PortalNetworkOpts) {
     // eslint-disable-next-line constructor-super
     super()
+    this.clientInfo = {
+      clientName: 'ultralight',
+      clientVersionAndShortCommit: `${packageJson.version}-${opts.shortCommit ?? ''}`,
+      operatingSystemAndCpuArchitecture: `${process.platform}-${process.arch}`,
```

Comment:
ideally, this is not something we should pull from a NodeJS builtin as it makes browser client stuff that much harder.  Can we just make this a required parameter to pass in to the constructor?